plugins/hanb.py

```
@@ -1,563 +1,272 @@
+
+    if not _categories:
+        categories = _get_all_categories(bot)
+        _categories.update(categories)
+
+    if not _categories:
+        return "No hanb boards loaded."
+
+    if text and text in _categories:
+        boards = _categories[text].get("boards", [])
+        if boards:
+            board = random.choice(boards)
```

Ruff (0.15.12)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)